che-theia-plugins.yaml

```diff
@@ -0,0 +1,179 @@
+cheTheiaPlugins:
```

Comment:
Just out of curiosity, is there any particular reason we use `cheTheiaPlugins` as the root object instead of just making the yaml document an array like:
```
- repository:
  url: https://github.com/microsoft/vscode-pull-request-github
  revision: 0.20.0
- repository:
  url: https://github.com/eclipse-cdt/cdt-gdb-vscode
  revision: 2cbbb857a2acf0b2e46bab30e0cbbfb547514856
- repository:
  url: https://github.com/clangd/vscode-clangd
  revision: c7f4a1b84fcc10d5b8241750cf2a84097ee4c37e
......
```

Reply:
Maybe if we need version field/ metadata later but non strong opinion or rename root element to a better one